cocos/core/math/utils.ts

```diff
@@ -245,8 +245,8 @@ export function inverseLerp (from: number, to: number, value: number) {
 /**
  * @en Compare the absolute values of all components and the component with the largest absolute value will be returned.
  * @zh 对所有分量的绝对值进行比较大小，返回绝对值最大的分量。
- * @param v 类 Vec3 结构
- * @returns 绝对值最大的分量
+ * @param v vec3 like value
+ * @returns max absolute component
```

Comment:
It is better to keep both English and Chinese.

Reply:
A large number of parameters are lack of Chinese notes, so it is suggested to change them uniformly later.